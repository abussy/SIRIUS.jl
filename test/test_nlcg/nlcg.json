{"EKS":-164.63161470296632,"F":-164.63167514644613,"entropy":-6.0443479813410666e-05,"eta":[{"ik":0,"ispn":0,"value":[-2.4901267486632923,-1.2476295835005131,-1.2396060041042396,-1.239606004098192,0.4636526138613269,0.6011039801014769,0.60110398010097,0.7038007075537163,0.7038007075539058,0.7214159461543738]},{"ik":0,"ispn":1,"value":[-2.590047206114662,-1.3445769178817866,-1.3374064650766502,-1.3374064650649289,0.4563898841451689,0.53429072079549,0.5342907207942206,0.6068665548302269,0.606866554830878,0.6441279150122212]},{"ik":1,"ispn":0,"value":[-2.489100072032021,-1.2505451971654045,-1.2439811611019138,-1.2399172023534089,0.5449321724870312,0.582262577133958,0.6125586376005866,0.7074063327901419,0.7145720821000835,0.8318018498156807]},{"ik":1,"ispn":1,"value":[-2.5891157831696874,-1.3471782966468482,-1.3413454499394548,-1.3376697138516345,0.4961527158675118,0.5202449796933396,0.5708848725803353,0.6113274262616524,0.6137848704604012,0.7870561999582377]},{"ik":2,"ispn":0,"value":[-2.491149021494177,-1.2406013855333287,-1.2392970163284553,-1.2392970163284598,0.3856355249102705,0.635044134320217,0.6350441343202007,0.6491937892764532,0.6788758618212829,0.7135138082394469]},{"ik":2,"ispn":1,"value":[-2.590975899204227,-1.338264069459721,-1.3371449949374297,-1.3371449949374283,0.38315492639979526,0.5618063919290237,0.5618063919290122,0.5728601200025514,0.5870434544258991,0.6141350860599611]},{"ik":3,"ispn":0,"value":[-2.4892846730121474,-1.25017237218368,-1.2429165179576653,-1.2394576565162374,0.5100283977064816,0.610213507272843,0.610316193819524,0.6824976034212257,0.7217158792990134,0.7364766939418367]},{"ik":3,"ispn":1,"value":[-2.589279138876731,-1.346848201765023,-1.3404230750375206,-1.3372686792383401,0.4785254858944506,0.5414921774795505,0.5527805168531874,0.5977188949900096,0.6371427533032765,0.6612429933753756]},{"ik":4,"ispn":0,"value":[-2.488442833865733,-1.2487316323691826,-1.2487316297252242,-1.240774881667438,0.5780711996681097,0.578071200468425,0.5969274125644453,0.707253261264596,0.707253261545594,0.9328998278777352]},{"ik":4,"ispn":1,"value":[-2.5885107057598367,-1.345600287350179,-1.3456002866382315,-1.3384793749416237,0.5188920842314447,0.5188920859064903,0.5347860457270533,0.6226864400527463,0.6226864405961066,0.8955078252453461]},{"ik":5,"ispn":0,"value":[-2.4877847332454457,-1.248192775464749,-1.246910825230422,-1.246910825228515,0.5455458097688768,0.5599046361936768,0.6932418012585133,0.6932418012582058,0.7174461545547293,0.8476202812618057]},{"ik":5,"ispn":1,"value":[-2.5879052506106275,-1.345119376838585,-1.3440168954701859,-1.3440168954697658,0.48958408492863903,0.492808100451252,0.6157954337223555,0.6157954337227339,0.6329749809281432,0.8145438667932187]}],"fermi_energy":0.6412327858953998,"fn":[{"ik":0,"ispn":0,"value":[1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.0,0.0,0.0]},{"ik":0,"ispn":1,"value":[1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.9999999999999988,0.9999999999999988,0.05239501163510418]},{"ik":1,"ispn":0,"value":[1.0,1.0,1.0,1.0,1.0,1.0,0.9999999999996476,0.0,0.0,0.0]},{"ik":1,"ispn":1,"value":[1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.9999999999998971,0.9999999999987991,0.0]},{"ik":2,"ispn":0,"value":[1.0,1.0,1.0,1.0,1.0,0.9979515997503993,0.9979515997504326,0.0003486839927788754,0.0,0.0]},{"ik":2,"ispn":1,"value":[1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.9999999999982954]},{"ik":3,"ispn":0,"value":[1.0,1.0,1.0,1.0,1.0,0.9999999999999662,0.9999999999999626,0.0,0.0,0.0]},{"ik":3,"ispn":1,"value":[1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.9835368196303366,2.040260405067329e-09]},{"ik":4,"ispn":0,"value":[1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.0,0.0,0.0]},{"ik":4,"ispn":1,"value":[1.0,1.0,1.0,1.0,1.0,1.0,1.0,0.9999999911807421,0.9999999911807373,0.0]},{"ik":5,"ispn":0,"value":[1.0,1.0,1.0,1.0,1.0,1.0,0.0,0.0,0.0,0.0]},{"ik":5,"ispn":1,"value":[1.0,1.0,1.0,1.0,1.0,1.0,0.9999999999910317,0.9999999999910317,0.9997408377550715,0.0]}],"ks_energy_comps":{"PAW_one_elec":-0.629235039201336,"PAW_total_energy":-38.07422318240108,"bxc":-0.11427914452661059,"entropy":-6.01341485273322e-05,"ewald":-86.00847149125877,"exc":-17.409002090666494,"valence_eval_sum":-8.324897945110113,"vha":66.63409418229067,"vxc":-17.75851291388753},"slope_eta":-1.0,"slope_x":-1.5924416117252522e-08,"step":0,"type":"cg_iteration"}{"EKS":-164.63161472293734,"F":-164.63167517756742,"entropy":-6.0454630086166274e-05,"fermi_energy":0.641233706050804,"ks_energy_comps":{"PAW_one_elec":-0.6292373250642551,"PAW_total_energy":-38.07422416149451,"bxc":-0.1143217251038495,"entropy":-6.0051371855459644e-05,"ewald":-86.00847149125877,"exc":-17.408957760627743,"valence_eval_sum":-8.325367728527816,"vha":66.63313655886611,"vxc":-17.758415648236433},"slope_eta":-1.0,"slope_x":-6.742506600840663e-09,"step":1,"type":"cg_iteration"}{"EKS":-164.63161476649265,"F":-164.63167518215246,"entropy":-6.041565982673662e-05,"fermi_energy":0.6412170165193231,"ks_energy_comps":{"PAW_one_elec":-0.6292342670833615,"PAW_total_energy":-38.07422447477512,"bxc":-0.11432893459755981,"entropy":-6.039247780971403e-05,"ewald":-86.00847149125877,"exc":-17.40903783592378,"valence_eval_sum":-8.324874002146247,"vha":66.63415710594305,"vxc":-17.758508388901888},"slope_eta":-1.0,"slope_x":-2.07484564670649e-09,"step":2,"type":"cg_iteration"}{"EKS":-164.63161478728085,"F":-164.63167518510514,"entropy":-6.039782427615843e-05,"fermi_energy":0.6412144199887598,"ks_energy_comps":{"PAW_one_elec":-0.629235260475622,"PAW_total_energy":-38.07422472533079,"bxc":-0.11434536311689031,"entropy":-6.035870261786603e-05,"ewald":-86.00847149125877,"exc":-17.40901796734059,"valence_eval_sum":-8.325066849162807,"vha":66.63376349175866,"vxc":-17.758467368098934},"slope_eta":-1.0,"slope_x":-2.2637340023282e-09,"step":3,"type":"cg_iteration"}{"EKS":-164.6316147984785,"F":-164.63167518690594,"entropy":-6.03884274324164e-05,"fermi_energy":0.6412125750450515,"ks_energy_comps":{"PAW_one_elec":-0.6292353466321015,"PAW_total_energy":-38.07422533490103,"bxc":-0.11435378095376376,"entropy":-6.034506836559948e-05,"ewald":-86.00847149125877,"exc":-17.409017800738333,"valence_eval_sum":-8.325101426379941,"vha":66.6336935549551,"vxc":-17.758458904691274},"slope_eta":-1.0,"slope_x":-9.042702859926438e-10,"step":4,"type":"cg_iteration"}