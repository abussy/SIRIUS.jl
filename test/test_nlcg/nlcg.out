nlcglib parameters
        T : 315.775
 smearing : Fermi-Dirac
   maxiter: 300
       tol: 1e-09
     kappa: 0.3
       tau: 0.1
   restart: 10
num electrons: 16
tol = 1e-09
Iteration      Free energy    	Residual       
0              -164.6316751464461	-1.5924416117253e-08 -1.0000000000000e+00
	 kT * S       : -0.0000604434798
	 Fermi energy : 0.6412327858954
	 KS energy    : -164.6316147029663
line search t_trial = 2.000000e-01
	 t_min = 3.648913e+00 q line prediction error: 2.06787831e-09 dE: 3.11212887e-08
line search took: 1.41638548e+00 seconds
steepest descent took: 9.37971400e-03 seconds
1              -164.6316751775674	-6.7425066008407e-09 -1.0000000000000e+00
	 kT * S       : -0.0000604546301
	 Fermi energy : 0.6412337060508
	 KS energy    : -164.6316147229373
line search t_trial = 2.00000000e-01
	 t_min = 1.48547815e+00 q line prediction error: -4.22886615e-10 dE: 4.58504701e-09
line search took: 1.41320883e+00 seconds
 CG gamma 3.361e-01
conjugated descent took: 1.355e-02 seconds
2              -164.6316751821525	-2.0748456467065e-09 -1.0000000000000e+00
	 kT * S       : -0.0000604156598
	 Fermi energy : 0.6412170165193
	 KS energy    : -164.6316147664926
line search t_trial = 2.000e-01
	 t_min = 3.946e+00 q line prediction error: -1.14104637e-09 dE: 2.95267455e-09
line search took: 1.41041862e+00 seconds
 CG gamma 1.344e+00
conjugated descent took: 1.377e-02 seconds
3              -164.6316751851051	-2.2637340023282e-09 -1.0000000000000e+00
	 kT * S       : -0.0000603978243
	 Fermi energy : 0.6412144199888
	 KS energy    : -164.6316147872809
line search t_trial = 2.000e-01
	 t_min = 1.388e+00 q line prediction error: 2.30215846e-10 dE: 1.80079951e-09
line search took: 1.40466037e+00 seconds
 CG gamma 2.669e-01
conjugated descent took: 1.328e-02 seconds
4              -164.6316751869059	-9.0427028599264e-10 -1.0000000000000e+00
	 kT * S       : -0.0000603884274
	 Fermi energy : 0.6412125750451
	 KS energy    : -164.6316147984785
kT * S   : -6.038842743242e-05
F        : -164.6316751869
KS-energy: -164.6316147985
NLCG SUCCESS
